doc/api/_types.rst

```diff
@@ -0,0 +1,18 @@
+**********************
+``matplotlib._types``
+**********************
+
+.. automodule:: matplotlib._types
```

Comment:
Ok for now.

When we get more entries, every entry should probably get it's own sub-page. But since we might fiddle with the organization/representation anyway (c.f. sphinx-gallery), it's not worth to work on this right now.